backend/tests/v5/helper/services.js

```diff
@@ -68,31 +68,28 @@ db.createModel = (teamspace, _id, name, props) => {
 	return DbHandler.insertOne(teamspace, 'settings', settings);
 };
 
-db.createRevision = (teamspace, modelId, { _id, author, timestamp, tag }) => {
-	const rev = {
-		_id: stringToUUID(_id),
-		author,
-		timestamp,
-		tag,
-	};
-	return DbHandler.insertOne(teamspace, `${modelId}.history`, rev);
+db.createRevisions = (teamspace, modelId, revisions) => {
+	const formattedRevisions = revisions.map((rev) => ({ ...rev, _id: stringToUUID(rev._id) }));
+	DbHandler.insertMany(teamspace, `${modelId}.history`, formattedRevisions);
```

Comment:
As above I think we should change this back to singular.

But for reference, you're missing a return here 
```suggestion
	return DbHandler.insertMany(teamspace, `${modelId}.history`, formattedRevisions);
```

insertMany returns a promise, and without returning that upstream, the caller would not know when it finishes.